frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -11,9 +11,11 @@ export type FaqCategory =
 export type Faq = {
   question: string;
   category: FaqCategory;
-  priority: number;
-  answerPreview: React.ReactNode;
   answerFull: React.ReactNode;
+  previewOptions?: {
```

Comment:
I feel mixed about calling this `previewOptions` but couldn't come up with an alternative. Do you have any preferences?

Reply:
hmm not really, i think it looks fine as-is though!